app/views/carts/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="container">
+
+
+  <% @items.each do |item| %>
+  <% quantity = @cart.contents[item.id.to_s].to_i %>
```

Comment:
This feels wroooong, but I couldn't figure it out any other way. Essentially, I want to grab all of the items from the cart and iterate through them. The problem is that the quantities are not in the @items array, but in the @cart.contents values, so I had to come up with this workaround. 

This is the part where I need the most help. I tried to iterate through the @cart.contents hash instead, but that was also not great. Let me know if we can come up with an alternative

Reply:
It might be a bit of an overthink, but if we wanted to change it we could do an each on the cart instead of a map and build a hash out with it. Building this as a model method might also be a possibility. Something like the below might work:
``` ruby
@items = {}
@cart.contents.each do |item, quantity|
   @items[Item.find(item)] = quantity
end
```

This would give us something we could iterate over that has item objects, and the quantities associated with them. I'd love to hear everyone's thoughts.

Reply:
Yes! That's a great solution! Let me try that on my branch and push it, unless someone else has another idea.

Reply:
@plapicola Just implemented that change, it worked great. Changing this from a draft to a pr